help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -0,0 +1,190 @@
+
+Once selected, or if you wish to skip this optional screen, select **[!UICONTROL Create]**
+
+### Enable Alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+### Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>* To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.
+
+Once the destination connection/instance has been created, you will be shown the audience activation flow. The following process walks through how to activate audiences using the Magnite Streaming Batch destination:
+
+#### Step 1: Select your new destination connection/instance
+
+![select the destination to activate audiences too](../../assets/catalog/advertising/magnite/destination-batch-active-audience-select-destination.png)
+
+#### Step 2: Select any/all audiences you want to activate
+
+![select the audiences to activate](../../assets/catalog/advertising/magnite/destination-batch-active-audience-select-audience.png)
+
+select **[!UICONTROL Next]** to continue.
+
+### Map attributes and identities {#map}
+
+After selecting which audience(s) to activate, you will need to map your device attributes and/or identities. The following continues the steps above for how to map attributes/identities in the Magnite Streaming Batch destination:
+
+#### Step 3: Map your source device ID to a magnite streaming device ID
+
+In the "SOURCE FIELD", you can select any attribute or identity for your devices. In this example, we've selected a custom IdentityMap called "DeviceId"
```

Comment:
Replace all mentions of SOURCE FIELD and TARGET FIELD with:

`**[!UICONTROL Source field]**`

`**[!UICONTROL Target field]**`